### DISCUSSION TITLE
CPU software AV1

### DISCUSSION THREAD

**Discussion by NightDarkness** [LizardByte/community#448] [Sunshine]
Are there plans to add software support for AV1 in the future? (Like svt-av1)

**Comment by CarloApri** [comment]
Yes, plesase.

**Comment by cgutman** [comment]
I answered this in a different discussion, but it's more relevant to write here:

No, AV1 is not supported for software encoding due to limitations of the SVT-AV1 encoder. I have filed a feature request to address the issue that's blocking us: https://gitlab.com/AOMediaCodec/SVT-AV1/-/issues/2120

There is also the matter of very poor performance. Even my Ryzen 9 7950X with AVX-512 support isn't powerful enough to deliver a playable frame rate using the low latency presets (which eliminates the possibility of frame-level multithreading). Even if it did work, the performance would probably be so bad that you couldn't actually use it for anything other than very low resolutions (below 1080p).
